### ISSUE TITLE
Ignore specific topics between two computers

### ISSUE THREAD

**Issue by Naraharirahul** [fkie/multimaster_fkie#126]
I'm trying to ignore a specific topic flowing from computer A to computer B. Is there any way of ignoring the topic apart from editing the computer B launch file?

**Comment by atiderko** [comment]
Currently there is no way to configure master_discovery on computer A to not provide topics to master_sync on computer B. At this time you must configure master_sync.launch on computer B.

**Comment by shervinkoushan** [comment]
@atiderko has anything been done in this area or is the status still the same?

**Comment by atiderko** [comment]
yes, there are three parameters *~hide_nodes*, *~hide_topics* and *~hide_services* for master_discovery. This should prevent the *master_sync* on another host from seeing them. However, the Node Manager can still display them.

**Comment by amilcarlucas** [comment]
Those parameters are not documented on http://wiki.ros.org/master_discovery_fkie?distro=melodic

And there is no http://wiki.ros.org/master_discovery_fkie?distro=noetic
